examples/langchain_example.py

```diff
@@ -1,61 +1,102 @@
 from typing import Any, Dict, List
 
 import boto3
+import newrelic.agent
 from langchain.agents import AgentType, initialize_agent
-from langchain.chat_models import ChatOpenAI
+from langchain.chat_models import ChatOpenAI, BedrockChat
+from langchain.llms import Bedrock
 from langchain.llms.base import LLM
-from langchain.llms.bedrock import Bedrock
 from langchain.tools import Tool
 
-from nr_openai_observability.langchain_callback import NewRelicCallbackHandler
+# When testing changes within the SDK, we need to load the changes from a local
+# directory. These lines allow for this. Unless you are testing changes to the
+# SDK itself, leave these lines commented out.
+#
+# Add vendor directory to module search path
+# import os
+# import sys
+# parent_dir = os.path.abspath(os.path.dirname(__file__))
+# vendor_dir = os.path.join(parent_dir, '../src')
 
+# sys.path.append(vendor_dir)
+# End adding SDK
```

Comment:
Should these lines be removed from the example?

Reply:
We could remove them. Just keep in mind that as we develop changes, putting these in again has been the quickest way for me to be able to test those changes short of pushing to a branch.

Reply:
I'm removing these now. Actually using poetry in my virtual environment has fixed that issue.